core/domain/exp_domain.py

```diff
@@ -1506,7 +1991,31 @@ def create_default_exploration(
             cls, exploration_id, title=feconf.DEFAULT_EXPLORATION_TITLE,
             category=feconf.DEFAULT_EXPLORATION_CATEGORY,
             objective=feconf.DEFAULT_EXPLORATION_OBJECTIVE,
+<<<<<<< HEAD
```

Comment:
Oh oh -- bad merge.

https://help.github.com/articles/resolving-a-merge-conflict-using-the-command-line/ -- see step 5. You'll need to fix it manually. I bet the backend tests don't pass.